applications/jonny5/doc/balance_crawler.md

```diff
@@ -0,0 +1,20 @@
+/*
+Section: Jonny5
+Title: Balance crawler
+Language: en-US
+*/
+
+# Balance crawler
+Disconnects active per_minute calls when account balance drops below zero.
+
+## Crawler configuration options
+Document - `system_config/jonny5`
+Options:
+- `balance_crawler_enabled` - boolean, default `true`. Enable crawler.
```

Comment:
I think the default should be false for now. Since this is new functionality, people should opt-in by manually enabling it. Once the kinks are worked out, we can discuss defaulting to true; for now I'd say default to false.